ocaml/xapi/cli_operations.ml

```diff
@@ -2690,6 +2690,7 @@ let snapshot_copy printer rpc session_id params =
 let snapshot_destroy printer rpc session_id params =
   let snap_uuid = get_snapshot_uuid params in
   let snap_ref = Client.VM.get_by_uuid rpc session_id snap_uuid in
+  if not (Client.VM.get_is_a_snapshot rpc session_id snap_ref) then failwith "This operation can only destroy VM snapshot.";
```

Comment:
There is grammar issue; it has to be either of these:

* This operation can only destroy *a* VM snapshot.
* This operation can only destroy VM *snapshots*.

Please:

* update the commit message such that it is clearly separated into a subject and a body. 
* do not force push over the current commit because this loses the discussion. We can always squash commits before merging.